.pre-commit-config.yaml

```
@@ -1,12 +1,12 @@
 repos:
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.3
+    rev: v4.0.0-alpha.8
     hooks:
       - id: prettier
         exclude: src/.*\.html
 
   - repo: https://github.com/psf/black
```

This mirror has mypyc-compiled wheels which are faster for pre-commit:
```suggestion
  - repo: https://github.com/psf/black-pre-commit-mirror
```